tests/acceptance/reminder-list-test.js

```diff
@@ -62,3 +62,13 @@ test('if there are no reminders there should be text "Please add your first remi
     assert.equal(find('.spec-no-reminder-status').text().trim(), 'Please add your first reminder!');
   });
 });
+
+test('if edit is selected a form is rendered where the user can update their reminder', function(assert){
+  server.createList('reminder', 1);
+
+  visit('/reminders/1/edit');
+
+  andThen(function() {
+    assert.equal(find('form').length, 1);
```

Comment:
Can we do more here? Assert the model.title starts off as the value in the input field. Afterwards change the input field and click save. Assert the current(url) is correct and the record is clean. 

So the acceptance tests in this file are pretty shallow. An acceptance test is the walk through of your UI. 
For example: If you expect the user to create a reminder on 'reminders/new'; an acceptance test asserts that the user can fill in the inputs and click submit. 

So can we beef some of these up before merging this?